### PR TITLE
{devel}[fosscuda/2020b] SoX v14.4.2 + torchaudio 0.9.0 w/ PyTorch 1.9.0

### PR DESCRIPTION
**Pull request by rarensu**
Some pretty simple easyconfigs.

### CODE REVIEW

**Comment by bedroge** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/t/torchaudio/torchaudio-0.9.0-fosscuda-2020b-PyTorch-1.9.0.eb [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+builddependencies = [
+    ('CMake', '3.18.4'),
+]
+
+dependencies = [
+    ('Python', '3.8.6'),
+    ('SciPy-bundle', '2020.11'),
+    ('PyTorch', local_pytorch_version),
+    ('SoX', '14.4.2'),
```
Comment (bedroge) [inline easybuild/easyconfigs/t/torchaudio/torchaudio-0.9.0-fosscuda-2020b-PyTorch-1.9.0.eb:34]: Looks like `torchaudio` now includes its own and patched version of `SoX`, which gets built if `BUILD_SOX=1`: https://github.com/pytorch/audio/tree/v0.8.1/third_party

> BUILD_SOX: Deteremines whether build and bind libsox in non-Windows environments. (no effect in Windows as libsox integration is not available) Default value is 1 (build and bind). Use 0 for disabling it.

So, my impression is that we no longer need to include `SoX` in the dependencies. Could you confirm this, and if this is indeed the case, remove it from the dependencies?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3902.accelgor.os - Linux RHEL 8.4, x86_64, AMD EPYC 7413 24-Core Processor (zen2), 1 x NVIDIA NVIDIA A100-SXM4-80GB, 470.82.01, Python 3.6.8
See https://gist.github.com/67192f62588119064f151a3e3b6fe26e for a full test report.

**Comment by boegel** [comment]
closing this since `foss/2020b` is deprecated, see https://docs.easybuild.io/policies/toolchains

**Pull request closed without merging**